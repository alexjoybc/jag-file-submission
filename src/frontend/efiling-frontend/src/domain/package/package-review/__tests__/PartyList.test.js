import React from "react";
import { render, waitFor } from "@testing-library/react";
import PartyList from "../PartyList";

describe("PartyList Component", () => {
  beforeEach(() => {});

  test("Matches snapshot", () => {
    const parties = [
      {
        partyType: "IND",
        roleType: "APP",
        firstName: "Bob",
        middleName: "Q",
        lastName: "Ross",
        roleDescription: "Applicant",
        partyDescription: "Individual",
      },
    ];
    const { asFragment } = render(<PartyList parties={parties} />);
    waitFor(() => {});

    expect(asFragment()).toMatchSnapshot();
  });
});
