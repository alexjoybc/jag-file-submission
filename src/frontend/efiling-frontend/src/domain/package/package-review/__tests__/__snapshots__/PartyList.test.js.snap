// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PartyList Component Matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="ct-party-list"
  >
    <div
      class="header"
    >
      <span
        class="d-none d-lg-inline col-lg-6"
      >
        Name
      </span>
      <span
        class="d-none d-lg-inline col-lg-3"
      >
        Party Role
      </span>
      <span
        class="d-none d-lg-inline col-lg-3"
      >
        Party Type
      </span>
    </div>
    <ul>
      <li>
        <span
          class="label col-sm-4 d-lg-none"
        >
          Name:
        </span>
        <span
          class="col-sm-8 col-lg-6"
        >
          Ross, Bob Q
        </span>
        <span
          class="label col-sm-4 d-lg-none"
        >
          Party Role:
        </span>
        <span
          class="col-sm-8 col-lg-3"
        >
          Applicant
        </span>
        <span
          class="label col-sm-4 d-lg-none"
        >
          Party Type:
        </span>
        <span
          class="col-sm-8 col-lg-3"
        >
          Individual
        </span>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
